_docs/style/colors.md

```diff
@@ -33,25 +155,35 @@ alerts, or data visualizations.
         {%endfor%}
       </div>
     {%endfor%}
-  </div>
+</div>
+{% endcolumn %}
+
+</div>
+
 </section>
 
+<section class="static-section" markdown="1">
 
-<section class="static-section"  markdown="1">
-### Alert Colors
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Object status
+
+Sometimes it is important to communicate the status of an entity, without also communicating urgency or criticality. For example, use orange 500 for a [Badge]({{site.baseurl}}/components/badge.html) or the New! tag to indicate that new items are available for review.
```

Comment:
[badge].... `New!` (New! is surrounded by single ticks.)

Reply:
not required per convo with Catherine.